Makefile

```diff
@@ -340,12 +340,17 @@ django-shell: .env
 DB_CACHE_DIR = db_cache/
 s3_key = $(db_name).sql.gz
 gz_file ?= $(DB_CACHE_DIR)$(s3_key)
+intermediary_file = sql_dump.sql
 
 load-db: $(DB_CACHE_DIR) $(gz_file) .env
 	@echo "Loading $(gz_file)"
-	@echo "drop database mc_dev; create database mc_dev;" | docker-compose run --rm web ./manage.py dbshell
-	@gzcat $(gz_file) | docker-compose run --rm web ./manage.py dbshell
-	@docker-compose run --rm web ./manage.py migrate
+	@echo "This will take a while, don't be alarmed if your console appears frozen."
+	@echo "drop database mc_dev; create database mc_dev; " > $(intermediary_file)
```

Comment:
If the user has a `sql_dump.sql` file already, we will end up deleting it.  Let's use `/tmp/` for temporary files.

Reply:
sure, I'm a bit worried about permissions. But I'll test locally to make sure iss fine.

Reply:
tis done, @pbx